src/packages.builds

```diff
@@ -59,11 +63,20 @@
       -->
       <PkgProjects Include="$(MSBuildThisFileDirectory)..\pkg\*\*.pkgproj" Exclude="$(MSBuildThisFileDirectory)..\pkg\*Private*\*.pkgproj" />
       <PkgProjects Include="*\pkg\**\*.pkgproj" />
+
+    </ItemGroup>
+
+    <ItemGroup>
```

Comment:
I feel like this should probably be the same in master... for future releases so that we don't have to port this on an on to all release branches... it should just work in master as we also define the set of projects to build, which are basically "all".